lab-examples/clos02/setup.sh

```diff
@@ -3,7 +3,7 @@
 CFG_DIR=./configs
 
 configure_SRL() {
-  OUT=$(gnmic -a clab-clos02-$1 --timeout 30s -u admin -p admin -e json_ietf --skip-verify set --update-path / --update-file $CFG_DIR/$1.yaml 2>&1)
+  OUT=$(gnmic -a clab-clos02-$1 --timeout 30s -u admin -p $PASSWORD -e json_ietf --skip-verify set --update-path / --update-file $CFG_DIR/$1.yaml 2>&1)
```

Comment:
it is fine to use the documented current password -- `NokiaSrl1!`

Reply:
okay I changed it to the documented password. Was not sure if you want to have it in git or not

Reply:
Maybe define the PASSWORD variable at the top.?!

Reply:
Good idea. I hope it looks good now.